packages/features/ee/payments/components/Payment.tsx

```diff
@@ -53,21 +65,24 @@ const PaymentForm = (props: Props) => {
 
     let payload;
     const params: { [k: string]: any } = {
-      uid: props.bookingUid,
+      uid: props.booking.uid,
       email: router.query.email,
     };
+
+    const returnUrl = getReturnUrl(props);
```

Comment:
`returnUrl` is already a string. We can just do

```suggestion
    const return_url = getReturnUrl(props);
```